src/rpc/net.cpp

```diff
@@ -456,12 +457,38 @@ static UniValue GetThinBlockStats()
         obj.push_back(Pair("validation_time", thindata.ValidationTimeToString()));
         obj.push_back(Pair("outbound_bloom_filters", thindata.OutBoundBloomFiltersToString()));
         obj.push_back(Pair("inbound_bloom_filters", thindata.InBoundBloomFiltersToString()));
+        obj.push_back(Pair("thin_block_size", thindata.ThinBlockToString()));
+        obj.push_back(Pair("thin_full_tx", thindata.FullTxToString()));
         obj.push_back(Pair("rerequested", thindata.ReRequestedTxToString()));
     }
     return obj;
 }
 // BitcoinUnlimited BUIP010 : End
 
+// BitcoinUnlimited BUIPXXX : Start
+static UniValue GetGrapheneStats()
+{
+    UniValue obj(UniValue::VOBJ);
+    bool enabled = IsGrapheneBlockEnabled();
+    obj.push_back(Pair("enabled", enabled));
+    if (enabled)
+    {
+        obj.push_back(Pair("summary", graphenedata.ToString()));
+        obj.push_back(Pair("inbound_percent", graphenedata.InBoundPercentToString()));
+        obj.push_back(Pair("outbound_percent", graphenedata.OutBoundPercentToString()));
+        obj.push_back(Pair("response_time", graphenedata.ResponseTimeToString()));
+        obj.push_back(Pair("validation_time", graphenedata.ValidationTimeToString()));
+        obj.push_back(Pair("filter", graphenedata.FilterToString()));
+        obj.push_back(Pair("iblt", graphenedata.IbltToString()));
+        obj.push_back(Pair("rank", graphenedata.RankToString()));
+        obj.push_back(Pair("graphene_block_size", graphenedata.GrapheneBlockToString()));
+        obj.push_back(Pair("graphene_additional_tx_size", graphenedata.AdditionalTxToString()));
+        obj.push_back(Pair("rerequested", graphenedata.ReRequestedTxToString()));
+    }
+    return obj;
```

Comment:
How about:
else obj.push_back(Pair("enabled", false);

Otherwise we return nothing which is confusing to interprete

Reply:
Wouldn't false already be pushed back from line 473?

Reply:
Ha, let this be a lesson to me... don't review code while in meeting :-(